sqlserver/datadog_checks/sqlserver/statements.py

```diff
@@ -274,7 +274,7 @@ def _load_raw_query_metrics_rows(self, cursor):
         self.log.debug("collecting sql server statement metrics")
         statement_metrics_query = self._get_statement_metrics_query_cached(cursor)
         now = time.time()
-        query_interval = self.collection_interval
+        query_interval = self.collection_interval * 2
         if self._last_stats_query_time:
             query_interval = now - self._last_stats_query_time
```

Comment:
This line will technically overwrite the prior `query_interval = self.collection_interval * 2` in most cases when `_last_stats_query_time` is set from prior run. Maybe `query_interval = max(query_interval, now - self._last_stats_query_time)`?

Reply:
Yea that's a good point.